docs/profiles.md

```diff
@@ -43,6 +43,21 @@ As of January 2025, there are three predefined profiles:
 * **backstage.io**: A simple configuration for a bare Backstage instance, utilizing the image available at https://github.com/backstage/backstage/pkgs/container/backstage.
 * **external**: A basis for third-party configurations external to the Backstage repository.
```

Comment:
Let's list rhdh profile in out-of-the-box along with other with link to ### The `rhdh` profile